### PR TITLE
[8.0][IMP][mail_tracking] Allow to define a custom mail_tracking base url

### PR DESCRIPTION
**Pull request by antespi**
This minor change allows to change the base url used for mail_tracking image, but if not defined it will fallbacks to `web.base.url`.
In some cases, this base URL must be different to `web.base.url`

### CODE REVIEW

File: mail_tracking/__openerp__.py
Comment (yajo) [inline mail_tracking/__openerp__.py:8]: Since this is a new feature, and not a fix, this should be `8.0.3.1.0` IMHO.

**Review by yajo — CHANGES_REQUESTED** [review]

File: mail_tracking/__openerp__.py
Comment (antespi) [reply mail_tracking/__openerp__.py:8]: No update is needed, and if you do not change anything in ir.config_parameter, then your addon will run exactly in the same way. That's why I use the last number.

Comment (yajo) [reply mail_tracking/__openerp__.py:8]: http://semver.org/#summary

**Review by yajo — APPROVED** [review]
Although I think versioning is incorrect, I'm not blocking this for that.

File: mail_tracking/__openerp__.py
Comment (pedrobaeza) [reply mail_tracking/__openerp__.py:8]: @Yajo, we use this syntax: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Review by pedrobaeza — APPROVED** [review]

File: mail_tracking/__openerp__.py
Comment (yajo) [reply mail_tracking/__openerp__.py:8]: Thanks for the pointer, although again it says that backwards-compatible new features increment `y`. This is the case.

However I'm not going to insist any more on this, it's not worth the effort, just do as you please :wink:

**Comment by pedrobaeza** [comment]
Fast-tracking as basic change

**Pull request merged**
